src/Core/Services/Implementations/OrganizationService.cs

```diff
@@ -1093,7 +1093,12 @@ private async Task SendInviteAsync(OrganizationUser orgUser, Organization organi
             throw new BadRequestException("User email does not match invite.");
         }
 
-        return await AcceptUserAsync(orgUser, user, userService);
+        var organizationUser = await AcceptUserAsync(orgUser, user, userService);
+
+        user.EmailVerified = true;
+        await _userRepository.ReplaceAsync(user);
```

Comment:
We should only do this if their email isn't already verified. Potentially save ourselves a db call.